js/main.js

```diff
@@ -99,6 +102,7 @@ function main(){
       var block = (aspenInfo.schedule.block);
       var blockSchedule = (aspenInfo.schedule.blockSchedule);
       var day = (aspenInfo.schedule.day);
```

Comment:
Instead of making another variable 'dy', just make 'day' declared in the global scope, and then set it on this line.